packages/models/src/lib/notifications/notification-kind.ts

```diff
@@ -0,0 +1,9 @@
+export enum NotificationKind {
```

Comment:
As far as I can figure out, these are all the categories of kinds of notifications. Am I missing anything here?